### ISSUE TITLE
A and B inputs should have "Handle 8 Positions"

### ISSUE THREAD

**Issue by awesomedoctorx3** [BEEmod/BEE2-items#1846]
so that the antlines will look nicer when connecting to death fizzlers.

**Comment by LautaroL20** [comment]
Good idea, same suggestion but for the AND Gates was maded by someone a time ago too.

**Comment by ZombiePwner1** [comment]
This will definitely make death fizzlers much nicer. @LautaroL20 Come on steam wheels.

**Comment by Luke18033** [comment]
These should instead be set to `handle_6_positions` to match fizzlers, then they can switch to an 8-way handle when that suggested stylevar is enabled.
